src/main/java/io/jenkins/plugins/opentelemetry/servlet/OpenTelemetryServletFilter.java

```diff
@@ -227,6 +235,36 @@ public void _doFilter(HttpServletRequest servletRequest, HttpServletResponse ser
             spanBuilder.setAttribute(SemanticAttributes.ENDUSER_ID, user.getId());
         }
 
+
+        ContextPropagators propagators = ContextPropagators.create(
```

Comment:
We should use `io.opentelemetry.sdk.OpenTelemetrySdk#getPropagators` which will require to expose this getter on our `io.jenkins.plugins.opentelemetry.OpenTelemetrySdkProvider`.

The default propagators of the OTel Auto Configure SDK are W3C Trace Context & Baggage, it's narower than your proposal that also includes OpenTracing and Jaeger propagators.
Ref https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#propagator

Reply:
Let me check how to auto-config the extractor.

Reply:
Well, done! ^-^